corehq/apps/hqwebapp/forms.py

```diff
@@ -44,6 +44,10 @@ def clean(self):
         if not password:
             raise ValidationError(_("Please enter a password."))
 
+        captcha = self.cleaned_data.get('captcha')
+        if settings.ENABLE_DRACONIAN_SECURITY_FEATURES and not captcha:
+            raise ValidationError(_("Please enter valid CAPTCHA"))
```

Comment:
@saiprakash-v 
this should be ? Please do also set without the setting set to make sure it does not break anything for other environments
```
if settings.ENABLE_DRACONIAN_SECURITY_FEATURES:
    if not self.cleaned_data.get('captcha'):  
        raise ValidationError(_("Please enter valid CAPTCHA"))
```

Reply:
Done 👍